Services/Models/DeviceModel.cs

```diff
@@ -144,22 +161,55 @@ public DeviceModelMessageSchema()
             }
         }
 
+        [JsonConverter(typeof(StringEnumConverter))]
         public enum DeviceModelMessageSchemaFormat
         {
+            [EnumMember(Value = "Binary")]
             Binary = 0,
+
+            [EnumMember(Value = "Text")]
             Text = 10,
+
+            [EnumMember(Value = "JSON")]
             JSON = 20
         }
 
+        [JsonConverter(typeof(StringEnumConverter))]
         public enum DeviceModelMessageSchemaType
         {
+            [EnumMember(Value = "Object")]
             Object = 0,
+
+            [EnumMember(Value = "Binary")]
             Binary = 10,
+
+            [EnumMember(Value = "Text")]
             Text = 20,
+
+            [EnumMember(Value = "Boolean")]
             Boolean = 30,
+
+            [EnumMember(Value = "Integer")]
             Integer = 40,
+
+            [EnumMember(Value = "Double")]
             Double = 50,
+
+            [EnumMember(Value = "DateTime")]
             DateTime = 60
         }
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        public enum DeviceModelType
+        {
+            [EnumMember(Value = "Undefined")]
```

Comment:
When can we have device model type as undefined ?

Reply:
The `DeviceModelType` is not mandatory for creating a new device model. The current support `DeviceModelType` in request are: 
* null
* string.empty
* "Custom"